CalculatorExercises/1-CalculatorBasic/Calculator.java

```diff
@@ -0,0 +1,43 @@
+import java.util.*;
+import java.lang.*;
+
+public class Calculator {
+   
+    private double result;
+
+    public double getResult() {
+        return result;
+    }
+
+    public Calculator () {
+        this.result = 0;
+    }
+
+    public void calculate (String input) {
+        try {
+                switch (input.charAt(0)) {
+                case '+':
+                    result += Double.valueOf(input.substring(1)); 
+                    break;
+                case '-':
+                    result -= Double.valueOf(input.substring(1));
+                    break;
+                case '/':
+                    result /= Double.valueOf(input.substring(1));
+                    break;
+                case '*':
```

Comment:
as a preparation to interfaces please add methods: add, substract, divide, multiply. You can call them here.
remember to mark them private if they are not used outside class